.github/workflows/main.yml

```diff
@@ -29,7 +32,21 @@ jobs:
           path: ~/.m2/repository
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-maven-
+      - name: Log into registry
```

Comment:
Hi @kamyarbb this will push a docker container for every build. I don't think this is the best way to do it.

Why not just add the (integration/system) tests to the service test route. It currently only contains normal test. If your test is marked with <TestName>IT.jave it will run during the verify step.